components/org.wso2.carbon.identity.handler.event.account.lock/src/main/java/org/wso2/carbon/identity/handler/event/account/lock/AccountLockHandler.java

```diff
@@ -471,6 +473,13 @@ protected boolean handlePostSetUserClaimValues(Event event, String userName, Use
                     log.debug("Error while reading Notification internally manage property in account lock handler", e);
                 }
             }
+            boolean isAdminInitiated;
+            if (IdentityUtil.threadLocalProperties.get().get(AccountConstants.ADMIN_INITIATED) != null) {
+                isAdminInitiated = (boolean) IdentityUtil.threadLocalProperties.get()
+                        .get(AccountConstants.ADMIN_INITIATED);
+            } else {
+                isAdminInitiated = true;
```

Comment:
No need for else part. When initialize isAdminInitiated, we can set it to true

Reply:
done